test/functional/nil_values_test.rb

```diff
@@ -14,35 +14,35 @@ def execute
       puts "value: #{params[option_name]}" if present
       HammerCLI::EX_OK
     end
-    
+
     build_options
   end
-  
+
   def assert_key_val(key, value, actual_output)
     assert_cmd(success_result(FieldMatcher.new(key, value.to_s)), actual_output)
   end
-  
+
   it 'accepts empty string' do
-    result = run_cmd(['--description=""'], {}, TestNilableCommand)
+    result = run_cmd(['--description=""'], { :verbosity => 1 }, TestNilableCommand)
```

Comment:
Could you please provide some default verbosity? It's less error prone + you get rid of passing it everywhere in the tests.